README.md

```diff
@@ -68,13 +69,13 @@ $ pip install -r requirements.txt
 ```
 
 The resulting hald images can be applied to any visuals in your application
-using GraphicsMagick bindings for Python, Ruby, PHP, JavaScript™, and other
+using GraphicsMagick bindings for Python, Ruby, PHP, JavaScript, and other
```

Comment:
Javascript is a trademark of Oracle. We should respect laws.
https://www.reddit.com/r/javascript/comments/8d0bg2/oracle_owns_javascript_so_apple_is_taking_down_my/

Reply:
Well, that would also mean each time you type in Photoshop, you'd insert the TM thing (and you aren't), so what do we do?